### DISCUSSION TITLE
能不能增加纯英文输入方案

### DISCUSSION THREAD

**Discussion by um-nyu** [imfuxiao/Hamster#519] [Q&A]
现在的中英文混合对双拼太头痛，而且很多时候只需要英语的时候，中英混在一起挺乱的，像下图这样的，（我不知道怎么导入）
![88855724-B602-4A73-84C9-3AAD3A83252B](https://github.com/imfuxiao/Hamster/assets/81240333/17d2ac1f-7370-40b5-a518-80405b9cfa22)

**Comment by imfuxiao** [comment]
建议做一个纯英文的输入方案，切换中文/英文输入方案使用。

**Comment by um-nyu** [comment]
已成功导入，谢谢
